api/advisory/db/crud.py

```diff
@@ -0,0 +1,39 @@
+""" Create/Read/Update/Delete """
+from datetime import date
+import logging
+from time import perf_counter
+from typing import List
+from sqlalchemy import select
+from sqlalchemy.engine.row import Row
+from sqlalchemy.ext.asyncio import AsyncSession
+from advisory.db.models.tileserver import (Hfi, FireZone)
+
+logger = logging.getLogger(__name__)
+
+
+async def get_hfi_area_percentages(session: AsyncSession, for_date: date) -> List[Row]:
+    """ DEPRECATED?
```

Comment:
Yes - pls. remove this file. It's not used by anything anymore.

Reply:
😆 you were the one who added the Deprecated? comment.

But yes, I'll delete this file.

Reply:
Yeah - deprecated and deleted :) That file has been removed, so you'd be re-introducing it.